app/assets/javascripts/vncapp.js

```diff
@@ -0,0 +1,31 @@
+/* global Screen, Client */
+(function () {
+  'use strict';
+  var client;
+
+$("#vncid").on('click', function() {
+    vncopen();
+  });
+
+
+  function vncopen() {
+var form = document.createElement("form");
+document.body.appendChild(form);
+form.method = "POST";
+form.action = "http://192.168.0.106:8090/vnc";
```

Comment:
@ranjithamca  This has a static ip ?  Can you include this as a property named `vnc_server: 192.168.0.106` in the nilavu.conf ?